kubernetes/README.md

```diff
@@ -3,7 +3,7 @@ No description provided (generated by Swagger Codegen https://github.com/swagger
 
 This Python package is automatically generated by the [Swagger Codegen](https://github.com/swagger-api/swagger-codegen) project:
 
-- API version: v1.12.1
+- API version: v1.12.2
```

Comment:
As discussed in https://github.com/kubernetes-client/python/pull/631#discussion_r221404512, currently Kubernetes latest release is 1.12.1. We are generating this client against the temporary 1.12.2 openapi spec file. I will file an issue in [gen repo](https://github.com/kubernetes-client/gen) to track the UX improvement.

/assign @yliaog